### PR TITLE
Rate limit updates

### PR DESCRIPTION
**Pull request by jakubvul-okta**
## Description:
- **What's changed?** Rate limits page update for DynamicScale
- **Is this PR related to a Monolith release?** No.

### Resolves:

* [OKTA-256030](https://oktainc.atlassian.net/browse/OKTA-256030)

### Screenshot

Screen Shot 2019-10-17 at 5 18 34 PM

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/rate-limits/index.md [hunk 1]
```diff
@@ -37,34 +37,70 @@ High capacity rate limits apply to a variety of endpoints across different APIs
...
+**Notes:**
+
+1. If Okta makes any change to the DynamicScale add-on's rate limits, such change will be communicated to customers via an updated version of this product documentation.
+2. Customers purchasing the DynamicScale add-on service will get best-effort additional protection beyond the multiplier they've purchased to handle any additional unforeseen spikes in Production:
+    1. This protection is not always guaranteed and should not be counted towards available rate limits
+    2. Additional protection availability is subject to infrastructure capacity available to your Org
+
+If your usage pattern exceeds the rate limits offered by DynamicScale add-on or the endpoint you're consuming isn't listed as part of the DynamicScale add-on then please contact your Okta Sales Representative regarding other options.
+
+For orgs that purchased the "High Capacity Rate Limit" SKU before 2019-10-24, the previous rate limits still apply.
```
Comment (karthicksolaimalai-okta) [inline packages/@okta/vuepress-site/docs/reference/rate-limits/index.md]: Should we add a link to the section where the old High Capacity Rate Limits will be moved to?  I think it makes sense.

**Comment by jakubvul-okta** [comment]
Updated:

Screen Shot 2019-10-18 at 4 17 05 PM

File: packages/@okta/vuepress-site/docs/reference/rate-limits/index.md [hunk 2]
```diff
@@ -37,35 +37,71 @@ High capacity rate limits apply to a variety of endpoints across different APIs
...
+
+* `/oauth2/{authorizationServerId}/v1`
+* `/oauth2/v1 except /oauth2/v1/clients`
+
+**SAML endpoints:**
+
+* `/app/template_saml_2_0/{key}/sso/saml`
+* `/app/{app}/{key}/sso/saml`
+
+**GET Users/Groups/Apps:**
```
Comment (prasanthmathialagan-okta) [inline packages/@okta/vuepress-site/docs/reference/rate-limits/index.md]: it is not just GETs. It includes updates and deletes as well. Does it make sense to rephrase it as "**Single User/Group/App operations (Get, Update and Delete)**"?

File: packages/@okta/vuepress-site/docs/reference/rate-limits/index.md [hunk 3]
```diff
@@ -37,35 +37,71 @@ High capacity rate limits apply to a variety of endpoints across different APIs
...
+* `/api/v1/apps/{id}`
+* `/api/v1/groups/{id}`
+* `/api/v1/users/{idOrLogin}`
+
+**Notes:**
+
+1. If Okta makes any change to the DynamicScale add-on's rate limits, such change will be communicated to customers via an updated version of this product documentation.
+2. Customers purchasing the DynamicScale add-on service will get best-effort additional protection beyond the multiplier they've purchased to handle any additional unforeseen spikes in Production:
+    1. This protection is not always guaranteed and should not be counted towards available rate limits
+    2. Additional protection availability is subject to infrastructure capacity available to your Org
```
Comment (prasanthmathialagan-okta) [inline packages/@okta/vuepress-site/docs/reference/rate-limits/index.md]: "subjected"

**Comment by bethwang-okta** [comment]
in "org-wide rate limits" section at top of page it also references High Capacity SKU which should be updated to DynamicScale: 
'High capacity rate limits apply to a variety of endpoints across different APIs for customers that purchased this add-on. These rate limits can be found below.'

File: packages/@okta/vuepress-site/docs/reference/rate-limits/index.md [hunk 4]
```diff
@@ -37,35 +37,71 @@ High capacity rate limits apply to a variety of endpoints across different APIs
...
 
-*The limits for these endpoints can be increased by purchasing the [High-Capacity add-on](#high-capacity-rate-limits).
-
 > **Note:** Public metadata endpoints for Org Authorization Server are: <br/> - `/oauth2/v1/keys` <br/> - `/.well-known/openid-configuration` <br/> Public metadata endpoints for Custom Authorization Servers are: <br/> - `/oauth2/{authorizationServerId}/v1/keys` <br/> - `/oauth2/{authorizationServerId}/.well-known/openid-configuration` <br/> - `/oauth2/{authorizationServerId}/.well-known/oauth-authorization-server`.
 
+### DynamicScale Rate Limits
+
+If your needs exceed Okta's default rate limits for the base product subscriptions (e.g. One App, Enterprise or IT Products) you've already purchased, you can purchase a "DynamicScale" add-on service which will grant you higher rate limits for the endpoints listed below. DynamicScale will increase your default rate limits by 3x to 1,000x, depending on the tier multiplier you purchase.  Customers can purchase this add-on annually for a Production tenant or temporarily for testing in a Sandbox tenant.  The following list of endpoints are included in the DynamicScale add-on service:
```
Comment (bethwang-okta) [inline packages/@okta/vuepress-site/docs/reference/rate-limits/index.md]: From the chat on friday the final  wording should be: DynamicScale will increase your default rate limits by 3x to 1000x, depending on the tier multiplier you purchase.

File: packages/@okta/vuepress-site/docs/reference/rate-limits/index.md (hunk 3 above)
Comment (jakubvul-okta) [reply packages/@okta/vuepress-site/docs/reference/rate-limits/index.md]: No, "subject" is correct.

File: packages/@okta/vuepress-site/docs/reference/rate-limits/index.md [hunk 5]
```diff
@@ -37,35 +37,71 @@ High capacity rate limits apply to a variety of endpoints across different APIs
...
+**Authentication endpoints:**
+
+* `/api/v1/authn`
+* `/api/v1/authn/factors/{factorIdOrFactorType}/verify`
+* `/api/v1/sessions`
+
+**OAuth2 endpoints:**
+
+* `/oauth2/{authorizationServerId}/v1`
+* `/oauth2/v1 except /oauth2/v1/clients`
```
Comment (prasanthmathialagan-okta) [inline packages/@okta/vuepress-site/docs/reference/rate-limits/index.md]: `/oauth2/v1` except `/oauth2/v1/clients`

**Pull request merged**